src/client/index.html

```diff
@@ -84,5 +84,7 @@
   </script>
   <% } %>
 
+  <script>require('electron-connect').client.create()</script> 
```

Comment:
Actually this will be best for development and production:
```
<% if (TARGET_DESKTOP && ENV !== 'prod') { %>
  <script>require('electron-connect').client.create()</script> 
  <% } %>

```